CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 1.43.8
```

Comment:
Since this is a big change, please make it 1.44.0 to make sure nothing auto-updates to this.

Reply:
It's no problem if clients update since this is strictly a refactor, no breaking changes.